bridge/package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Token Bridge Contracts",
   "scripts": {
     "preinstall": "npx npm-force-resolutions",
-    "ganache": "npx ganache-cli -i 5777 -k istanbul -v",
-    "ganache-mirror": "npx ganache-cli -i 5776 -p 8546 -k istanbul -v",
+    "ganache": "npx ganache-cli --quiet -i 5777 -k istanbul -v",
+    "ganache-mirror": "npx ganache-cli --quiet -i 5776 -p 8546 -k istanbul -v",
```

Comment:
I added this because at least in development it seems to be unnecessary to have a constant log output while testing (eats up a lot of RAM) - this might use less resources when running the CI as well